types/pouchdb-find/index.d.ts

```diff
@@ -96,6 +96,9 @@ declare namespace PouchDB {
 
             /** Number of docs to skip before returning. */
             skip?: number;
+
+            /** Set which index to use for the query. It can be “design-doc-name” or “[‘design-doc-name’, ‘name’]”. */
+            use_index?: string;
```

Comment:
You should probably cover the array of string case, too (especially since it is mentioned in the doc comment here):

```ts
use_index?: string | Array<string>;
```